paris/src/testDebug/java/com/airbnb/paris/utils/TypefaceEquals.kt

```diff
@@ -0,0 +1,23 @@
+package com.airbnb.paris.utils
+
+import android.graphics.Typeface
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNull
+import org.robolectric.Shadows.shadowOf
+
+/**
+ * ShadowTypeface should be used for Typeface comparison; otherwise false positives will be reported.
+ */
+fun assertTypefaceEquals(expected: Typeface?, actual: Typeface?) {
+    if (expected == null) {
+        assertNull(actual)
+        return
+    }
+
+    val expectedShadow = shadowOf(expected)
+    val actualShadow = shadowOf(actual)
```

Comment:
As someone not very familiar with Robolectric shadows this is a bit of a puzzler at first glance. Could you leave some brief comments explaining what goes on?